GirafIntegrationTest/tests/test_authorization.py

```diff
@@ -0,0 +1,381 @@
+from requests import get, post, put, delete
+from testlib import order, BASE_URL, GIRAFTestCase
+
+
+class TestAuthorization(GIRAFTestCase):
+    """
+    Testing API requests without authorization
+    """
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        """
+        Setup necessary data and state when class is loaded
+        """
+        super(TestAuthorization, cls).setUpClass()
+        print(f'file:/{__file__}\n')
+        cls.EXPIRED_TOKEN = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiI1YmM0YzAzMC1mOGQxLTRhYTAtOTBlOC05MTNh" \
+                            "MDYwOTA4YWIiLCJodHRwOi8vc2NoZW1hcy54bWxzb2FwLm9yZy93cy8yMDA1LzA1L2lkZW50aXR5L2NsYWltc" \
+                            "y9uYW1laWRlbnRpZmllciI6Ijg0MTJkOTk1LWIzODEtNGY4My1iZDI1LWU5ODY2NzBiNTdkOSIsImV4cCI6MT" \
+                            "UyNTMwMzQyNSwiaXNzIjoibm90bWUiLCJhdWQiOiJub3RtZSJ9.8KXRRqF3B5s8tUki7u5j0TqK-189QIpApd" \
+                            "OC6aSxOms"
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        """
+        Remove or resolve necessary data and states after class tests are done
+        """
+        super(TestAuthorization, cls).tearDownClass()
+
+    def setUp(self) -> None:
+        """
+        Setup necessary data and states before each test
+        """
+        pass
+
+    def tearDown(self) -> None:
+        """
+        Remove or resolve necessary data and states after each test
+        """
+        pass
+
+    """
+    Account endpoints
+    """
+    @order
+    def test_auth_POST_account_set_password(self):
```

Comment:
For all the following tests: I think the names could be better and more descriptive :)